dask/dataframe/io/io.py

```diff
@@ -146,6 +146,40 @@ def from_array(x, chunksize=50000, columns=None, meta=None):
     return new_dd_object(dsk, name, meta, divisions)
 
 
+def from_dict(data, npartitions, orient="columns", dtype=None, columns=None):
```

Comment:
I see we're adding this as a top-level `dd.from_dict` method, but the corresponding pandas method is actually a classmethod -- so `pd.DataFrame.from_dict` instead of `pd.from_dict`. My default is usually to match the existing DataFrame API if one exists. Thoughts of making this `dd.DataFrame.from_dict`?